llvm/lib/Transforms/Scalar/SROA.cpp

```diff
@@ -116,8 +116,11 @@ STATISTIC(NumVectorized, "Number of vectorized aggregates");
 static cl::opt<bool> SROAStrictInbounds("sroa-strict-inbounds", cl::init(false),
                                         cl::Hidden);
 
-static cl::opt<bool> SROAAvoidAggregates("sroa-avoid-aggregates", cl::init(false),
-                                         cl::Hidden);
+/// Prevent SROA from desegregating structs containing variable width integers
```

Comment:
```suggestion
/// Prevent SROA from desegregating structs containing arbitrary-width integers
```
There is no variation over time in the width itself.